app/controllers/events/email.py

```diff
@@ -22,3 +23,10 @@ def fetchEmailLogData(eventId):
         return {'last_log': "The last email was sent to " + last_email.recipientsCategory + " on " + last_email.dateSent.strftime('%m/%d/%Y') + " by " + last_email.sender.email  + "." , 'last_log2': " Subject: " + last_email.subject}
     else:
         return {'exists': False}
+
+@main_bp.route("/getProgramSender/", methods=['POST'])

```

Comment:
This also needs to be switched to GET if the ajax call is using GET. You should get your data with `request.args.get("programId")` instead of with `request.form`